integration-tests/mongodb-panache/src/test/java/io/quarkus/it/mongodb/panache/resolvers/ResolverEntityWithDatabase.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.it.mongodb.panache.resolvers;
+
+import io.quarkus.mongodb.panache.common.MongoEntity;
+
+public class ResolverEntityWithDatabase {
+    public static final String DATABASE_NAME = "database-from-anotation";
+
+    @MongoEntity(database = DATABASE_NAME)
+    public static class Entity {
```

Comment:
Why isn't this just a top level class?  I know this is "just" test code but there doesn't seem to be a reason for the nesting here.

Reply:
The idea is use the constant in test assertations to avoid passing a hard coded string in assertation, but If you thinks that it is not a worth reason I can fix it.

Reply:
That constant can still be used without having to nest like that.